src/agent/sqlite_manager/tests/sqlite_manager_test.cpp

```diff
@@ -0,0 +1,16 @@
+#include "gtest/gtest.h"
+class SQLiteManagerTest : public ::testing::Test
+{
+protected:
+    const std::string dbName = "testdb.db";
+
+    void SetUp() override {}
+
+    void TearDown() override {}
+};
+
+TEST_F(SQLiteManagerTest, OneTest)
+{
+
+    EXPECT_EQ(2, 2);
```

Comment:
A more meaningful test at this moment would be to create an instance of the `SQLiteManager` class, this way we are sure the it is not optimized away for not being used.

Reply:
Done